packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-design.md

```diff
@@ -0,0 +1,54 @@
+---
+id: Design for accessibility
+section: accessibility
+---
+
+As described in our [accessibility fundamentals](/accessibility/accessibility-fundamentals), users may interact with your product through a variety of assistive technologies. In addition to [developing for accessibility](/accessibility/accessibility-development), you must also incorporate it into your product's UI design. 
+
+The following table differentiates the accessibility responsibilities that are up to designers, developers, or both to uphold.
+
+| Designer | Designer and developer | Developer |
+| --- | --- | --- |
+| Color  | Layout and structure  | Semantic HTML  |
+| Contrast  | Information architecture and navigation | Tab order  |
+| Content  | Interaction patterns | Bypass blocks |
+
+## Color and contrast
+
+Color and contrast go hand-in-hand for visual design accessibility. 
+
+For example, you should consider contrast when using colors commonly affected by color perception deficiency ([such as those described by W3C](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Overview_of_Low_Vision#Color_Perception)). Different colors in a UI should be distinguishable for users across the color perception deficiency spectrum, typically by ensuring that each color has a different contrast weight. 
```

Comment:
Nit: could we update the link text here? Just out of context, there's the question of "what are those?" in regards to the "such as those..." text. Maybe wrapping the link around the "color perception" text instead, or "(as described in [color perception by W3C](...)"?